docs/styleguide/PyGuide.md

```diff
@@ -37,6 +37,8 @@
   - [6.3 Folder structure](#63-folder-structure)
   - [6.4 Test runtime considerations](#64-test-runtime-considerations)
   - [6.5 BKC management](#65-bkc-management)
+- [7 SDL rules](#s7-sdl-rules)
```

Comment:
I don't think external developers know what is SDL. Either explain the abbreviation or just call it "Security rules"